lib/launcher.js

```diff
@@ -55,13 +55,20 @@ class Launcher {
     async installDependencies () {
         info('Installing project dependencies')
         if (this.config.moduleCache) {
+            info('Using module_cache')
             const sourceDir = path.join(this.config.dir, 'module_cache/node_modules')
+            const targetDir = path.join(this.projectDir, 'node_modules')
             try {
                 await fs.access(sourceDir)
             } catch (ee) {
                 return Promise.reject(ee)
             }
-            return fs.symlink(sourceDir, path.join(this.projectDir, 'node_modules'), 'dir')
+
+            if (existsSync(targetDir)) {
+                await fs.unlink(targetDir)
+            }
```

Comment:
```suggestion
            if (existsSync(targetDir)) {
                await fs.rm(targetDir, { force: true, recursive: true })
            }
```
If `node_modules` exists and is a real directory, `unlink` will throw an error. That is a very possible scenario if the user enables the module cache after running with it.